package-lock.json

```diff
@@ -21887,13 +21942,25 @@
         "set-value": "^2.0.1"
       },
       "dependencies": {
+        "extend-shallow": {
+          "version": "2.0.1",
+          "resolved": "https://registry.npmjs.org/extend-shallow/-/extend-shallow-2.0.1.tgz",
+          "integrity": "sha1-Ua99YUrZqfYQ6huvu5idaxxWiQ8=",
+          "dev": true,
+          "requires": {
+            "is-extendable": "^0.1.0"
+          }
+        },
         "set-value": {
-          "version": "4.1.0",
-          "resolved": "https://registry.npmjs.org/set-value/-/set-value-4.1.0.tgz",
-          "integrity": "sha512-zTEg4HL0RwVrqcWs3ztF+x1vkxfm0lP+MQQFPiMJTKVceBwEV0A569Ou8l9IYQG8jOZdMVI1hGsc0tmeD2o/Lw==",
+          "version": "2.0.1",
```

Comment:
This shouldn't be reverted back to the pre-security-fix state. Perhaps run the following:

```
git checkout master -- package-lock.json
npm install
```

That should fix this up.

Reply:
Done.